src/mixins/amalgamation-mixin.ts

```diff
@@ -91,9 +91,9 @@ export default class AmalgamationMixin extends Vue {
     return null
   }
 
-  /** Disallow if foreign into ULC if there is also a limited company. */
+  /** Disallow if foreign and there is also a BC company, into ULC. */
```

Comment:
'A foreign corporation must not amalgamate with a BC Company and continue as an Unlimited Liability Company.'